### PR TITLE
Feature/404 qts mvp

### PR DESCRIPTION
**Pull request by warrensearle**

### CODE REVIEW

File: src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+              :key="i"
+            >
+              {{ option.answer }}
+            </li>
+          </ol>
+        </dd>
+      </div>
+    </dl>
+
+    <button
```
Comment (crablab) [inline src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue:142]: The flow on this page is a bit odd. You end up here having completed setting up your QT, and the only option is then "submitForApproval". 

I wonder if this button is in the wrong place and should be on the dashboard? We should at least have a "go back to dashboard" or the like button as you feel a little trapped!

File: src/views/Exercises/Tasks/QualifyingTests/Cover.vue [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    </Table>
+
+    <button
+      class="govuk-button govuk-!-margin-right-3"
+      @click="btnCreate"
+    >
+      Create New
+    </button>
+
+    <div class="separator">
```
Comment (lloback) [inline src/views/Exercises/Tasks/QualifyingTests/Cover.vue:36]: we eventually need to remove this temporary navigation from here ....

File: src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue (hunk 1 above)
Comment (lloback) [reply src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue:142]: @crablab is this a rhetorical question, or is it preventing the code from being merged? This is the type of things that we do our best and wait for feedback from Claire / Becky / others.

**Review by lloback — APPROVED** [review]
In my humble opinion the code is ready to merge - there are still some TODO lists. but it is what it is!

File: src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue (hunk 1 above)
Comment (crablab) [reply src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Review.vue:142]: @lloback It was more an observation. I asked Warren (https://judicial-appointments.slack.com/archives/G011STFNTJB/p1598354692028200) where I should put non-critical things like this, and he suggested in the review.

Happy to move it to the wash up ticket though, if you prefer, as it's very much non-critical.

**Review by crablab — APPROVED** [review]

File: src/views/Exercises/Tasks/QualifyingTests/Cover.vue (hunk 2 above)
Comment (warrensearle) [reply src/views/Exercises/Tasks/QualifyingTests/Cover.vue]: Ooh, just did this

**Pull request merged**